### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by dimakis**
Just a general refactor including

- updated comments
- fixed typos
- changing of URLs to constants
- addition of extra general exception handling
- updated the run.yml for integration tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,4 +20,14 @@ This collection requires some additional dependencies to be installed:
...
 pip install rhoas-sdks --force-reinstall
 ```
-All modules require an 'OFFLINE_TOKEN' environment variable to be set with a valid token. This token is used to authenticate the user. The token is an OpenShift Cluster Manager API Token and can be found here https://console.redhat.com/openshift/token
\ No newline at end of file
+
+## Using collection
+
+All modules require an 'OFFLINE_TOKEN' environment variable to be set with a valid token. This token is used to authenticate the user. The token is an OpenShift Cluster Manager API Token and can be found [here](https://console.redhat.com/openshift/token)
+
+Two further environment variables are required to be set for the collection to work. These environment variables can be placed in a `.env` file. They are:
```
Comment (wtrocki) [inline README.md:28]: Required? They are only for internal needs. No one would use them outside our team.

File: README.md [hunk 2]
```diff
@@ -20,4 +20,14 @@ This collection requires some additional dependencies to be installed:
 ```shell
 pip install rhoas-sdks --force-reinstall
 ```
-All modules require an 'OFFLINE_TOKEN' environment variable to be set with a valid token. This token is used to authenticate the user. The token is an OpenShift Cluster Manager API Token and can be found here https://console.redhat.com/openshift/token
\ No newline at end of file
+
+## Using collection
+
+All modules require an 'OFFLINE_TOKEN' environment variable to be set with a valid token. This token is used to authenticate the user. The token is an OpenShift Cluster Manager API Token and can be found [here](https://console.redhat.com/openshift/token)
```
Comment (wtrocki) [inline README.md:26]: This is not secure also will not work on external nodes. Can we have this as argument?

File: README.md (hunk 1 above)
Comment (wtrocki) [inline README.md:28]: This section is more for contributing.md

**Review by wtrocki — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (dimakis) [reply README.md:26]: As in prompt the user to specify the `OFFLINE_TOKEN` when running the playbook?

Comment (wtrocki) [reply README.md]: Token needs to be argument.

**Review by wtrocki — APPROVED** [review]

**Pull request merged**